src/Project.php

```diff
@@ -27,6 +28,9 @@ public static function fromArray(array $in): self
         $project->updatedTimestamp = $in['updatedTimestamp'] ?? '';
         $project->mlflowServerVersion = $in['mlflowServerVersion'] ?? '';
         $project->mlflowServerVersionLatest = $in['mlflowServerVersionLatest'] ?? '';
+        $project->persistentStorage = array_key_exists('persistentStorage', $in)
```

Comment:
Tady bych volil asi spis `isset` misto `array_key_exists`. Podud by ti prislo `persistentStorage: null`, tak to padne, protoze `PersistentStorage::fromArray` ocekava pole.

Reply:
Jo, teraz uz tam moze byt isset asi, este sa nad tym zamyslim 🤔